### PR TITLE
Added slaves field in executor CRD to add slaves configuration

### PR DESCRIPTION
**Pull request by hiteshwani**
## Pull request description 

Added slaves field in executor CRD to add slaves configuration

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [x] tested locally
- [x] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: config/crd/bases/tests.testkube.io_testsuites.yaml [hunk 1]
```diff
@@ -356,6 +356,18 @@ spec:
                     description: test kubernetes namespace (\"testkube\" when not
                       set)
                     type: string
+                  runningContext:
```
Comment (vsukhin) [inline config/crd/bases/tests.testkube.io_testsuites.yaml:359]: exclude this file from pr. it was done by purpose

File: config/crd/bases/tests.testkube.io_testsources.yaml [hunk 2]
```diff
@@ -70,6 +70,9 @@ spec:
                       name:
                         description: object name
                         type: string
+                      namespace:
```
Comment (vsukhin) [inline config/crd/bases/tests.testkube.io_testsources.yaml:73]: exclude this file from pr. it was done by purpose

**Review by vsukhin — APPROVED** [review]
good in general, just exclude files you didn't change

File: config/crd/bases/tests.testkube.io_testsuites.yaml (hunk 1 above)
Comment (hiteshwani) [reply config/crd/bases/tests.testkube.io_testsuites.yaml]: Discarded changes from file

File: config/crd/bases/tests.testkube.io_testsuiteexecutions.yaml [hunk 3]
```diff
@@ -305,8 +305,8 @@ spec:
                                           type: string
                                         type: array
                                       omitFolderPerExecution:
-                                        description: don't use a separate folder for execution
-                                          artifacts
+                                        description: don't use a separate folder for
+                                          execution artifacts
```
Comment (dejanzele) [inline config/crd/bases/tests.testkube.io_testsuiteexecutions.yaml:309]: why did it break a new line here?

File: config/crd/bases/executor.testkube.io_webhooks.yaml [hunk 4]
```diff
@@ -71,7 +71,8 @@ spec:
                 description: Labels to filter for tests and test suites
                 type: string
               uri:
-                description: Uri is address where webhook should be made (golang template supported)
+                description: Uri is address where webhook should be made (golang template
+                  supported)
```
[the remark this answers is not in the document]
Comment (parauliya) [reply config/crd/bases/executor.testkube.io_webhooks.yaml]: Done

**Review by dejanzele — APPROVED** [review]

**Pull request merged**